src/flb_output_thread.c

```diff
@@ -203,6 +204,9 @@ static void output_thread(void *data)
      */
     flb_engine_evl_set(th_ins->evl);
 
+    /* Create the bucket queue (FLB_ENGINE_PRIORITY_COUNT priorities) */
+    evl_bktq = mk_bucket_queue_create(FLB_ENGINE_PRIORITY_COUNT);
```

Comment:
Not deleting this memory for now. Move pqueue to thread instance config potentially handle delete in this engine function in cleanup section.